.mergify.yml

```diff
@@ -0,0 +1,18 @@
+pull_request_rules:
+  - name: automatic merge when GitHub branch protection passes + explicit two approvals
+    conditions:
+      - "#approved-reviews-by>=1"
```

Comment:
Only one required review?

Reply:
yes. We shortly discussed this today already: As os-autoinst has less active reviewers we should not need more than one approval (for now).